lib/elixir/lib/module.ex

```diff
@@ -749,11 +765,15 @@ defmodule Module do
       end
 
   """
-  def defines?(module, tuple, kind) do
+  @spec defines?(module, function_arity, def_kind) :: boolean
+  def defines?(module, {function_macro_name, arity} = tuple, def_kind)
+      when is_atom(module) and is_atom(function_macro_name) and
+           is_integer(arity) and arity >= 0 and arity <= 255 and
```

Comment:
Nope. They wouldn't work at this point. Ranges work different than lists with `in`.